lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+  padding-right: $nx-spacing-xs;
+}
+
+.nx-tag--selectable {
+  cursor: pointer;
+  display: inline-flex;
```

Comment:
I don't think flex is necessary here.  The icon and text use the same font size so they should align just fine using inline layout.  You've got them as `<div>`s right now though, so you'd need to change them to `<span>`s, which I think makes sense.

Reply:
> I don't think flex is necessary here. The icon and text use the same font size so they should align just fine using inline layout. You've got them as `<div>`s right now though, so you'd need to change them to `<span>`s, which I think makes sense.

It is though, without it and `align-items: centre;` my vertical alignment within the tag go all wonky, and the truncation doesn't work right.